README.md

```diff
@@ -1,22 +1,22 @@
 # [WebF](https://openwebf.com/) [![pub package](https://img.shields.io/pub/v/webf.svg)](https://pub.dev/packages/webf)
 
-WebF (Web on the Flutter) is a W3C standard compliant Web rendering engine based on Flutter, it can run web application on Flutter natively.
+WebF (Web on Flutter) is a W3C standards compliant Web rendering engine based on Flutter, it can run web applications on Flutter natively.
```

Comment:
There are another project called `Flutter on the web`, so why `Web on the Flutter` is wrong?

Reply:
I don't have a very certain answer. But based on my experiences normally we don't use "the Flutter" directly. There are some cases we use "the" according to https://group.langlib.com/Thread/Detail/0866-1249922 , where we say "the sun" the "the rain" for the unique things("the web" is like this), and "the cinema" the "the opera" to specify a special target from many. "Flutter" does not fill either of them.

If you read wiki https://en.wikipedia.org/wiki/World_Wide_Web , "the web" is frequently used. And for Flutter https://en.wikipedia.org/wiki/Flutter_(software) , it's alway "the Flutter Interactive event" "the Flutter team", "The Flutter framework" to specify a "framework" from many, rather than using "the Flutter". 

Another small example is the sentence in the mentioned wiki, it used "similar to React" rather than "similar to the React", and as well we normally don't say "the React".

> A widget describes the logic, interaction, and design of a UI element with an implementation similar to React.

Maybe also compare usages on Google:

* [on flutter](https://www.google.com/search?q=%22on+flutter%22+site%3Ahttps%3A%2F%2Fflutter.dev%2F)
* [on the flutter](https://www.google.com/search?q=%22on+the+flutter%22+site%3Ahttps%3A%2F%2Fflutter.dev%2F)

So I supposed "for Flutter" and "on Flutter" are preferred. But still I'm not totally sure. Advices from native English speakers would be better.